packages/at_commons/lib/src/keystore/public_key_hash.dart

```diff
@@ -0,0 +1,30 @@
+/// Represents hash of an atsign's public encryption key and the hashing algorithm used
+class PublicKeyHash {
+  String? hash;
+  PublicKeyHashingAlgo? publicKeyHashingAlgo;
```

Comment:
Why are these nullable?

Reply:
field should not be nullable. I have removed nullable fields